models/zigzag/arbitrum/zigzag_v1_arbitrum_trades.sql

```diff
@@ -0,0 +1,106 @@
+{{ config(
+    schema = 'zigzag_v1_arbitrum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "zigzag_v1",
+                                \'["mtitus6"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2022-09-01' %}
+
+with
+  dexs as (
+    select
+      call_block_time as block_time,
+      zzmo.makerOrder:sellToken as token_sold_address,
+      zzmo.makerOrder:buyToken as token_bought_address,
+      zzmo.output_matchedFillResults:takerSellFilledAmount as token_bought_amount_raw,
+      zzmo.output_matchedFillResults:makerSellFilledAmount as token_sold_amount_raw,
+      NULL AS amount_usd,
+      zzmo.makerOrder:user as maker,
+      zzmo.takerOrder:user as taker,
+      call_tx_hash as tx_hash,
```

Comment:
Hi @mtitus6 , I have reviewed the code, and overall it seems legitimate to me.
I am not sure why the `dbt slim ci` test have not been triggered.
I am guessing is because of the advanced sql syntax you are using here(get attributes out of a json).
So here is the (uniqueness test)[https://dune.com/queries/1517700] I manually ran

Reply:
I am not sure either.  

Is there something I need to change or should I just remain patient with the existing code?

Reply:
hi both -- please ensure to merge dune main branch into working branch(s) to keep up-to-date. i pushed a commit with the merge of main and the tests are now running